[bug] Can not download in front end

### Describe the bug

in the browser, we can use <a> like this:
```js
 const eleLink = document.createElement("a");
                eleLink.href = url;
                eleLink.download = `${title} - ${artist}`;
                eleLink.style.display = "none";
                document.body.appendChild(eleLink);
                eleLink.click();
                document.body.removeChild(eleLink);
```
to download file in front; but in tauri,this function can not take effect.

### Reproduction

_No response_

### Expected behavior

_No response_

### Platform and versions

```shell
› OS: Mac OS 12.4.0 X64
  › Node.js: 16.14.0
  › npm: 8.3.1
  › pnpm: 6.32.3
  › yarn: 1.22.17
  › rustup: 1.24.3
  › rustc: 1.61.0
  › cargo: 1.61.0
  › Rust toolchain: stable-aarch64-apple-darwin
```


### Stack trace

_No response_

### Additional context

_No response_

> yep. If you want to keep configuring the window in tauri.conf.json you can add `create: false` to the window config (requires latest tauri version) and then do something like this
> ...

Unfortunately, your example is not working:

```
error[E0599]: no method named `on_download` found for enum `Result` in the current scope
   --> src/main.rs:15:16
    |
14  |               let webview_window = WebviewWindowBuilder::from_config(handle, &app.config().app.windows.get(0).unwrap().clone())
    |  __________________________________-
15  | |               .on_download(|webview, event| {
    | |               -^^^^^^^^^^^ method not found in `Result<WebviewWindowBuilder<'_, Wry<EventLoopMessage>, AppHandle>, Error>`
    | |_______________|
    |
    |
note: the method `on_download` exists on the type `WebviewWindowBuilder<'_, tauri_runtime_wry::Wry<EventLoopMessage>, AppHandle>`
   --> C:\Users\Docker\.cargo\registry\src\index.crates.io-6f17d22bba15001f\tauri-2.2.2\src\webview\webview_window.rs:301:3
    |
301 | /   pub fn on_download<F: Fn(Webview<R>, DownloadEvent<'_>) -> bool + Send + Sync + 'static>(
302 | |     mut self,
303 | |     f: F,
304 | |   ) -> Self {
    | |___________^
help: use the `?` operator to extract the `WebviewWindowBuilder<'_, tauri_runtime_wry::Wry<EventLoopMessage>, AppHandle>` value, propagating a `Result::Err` value to the caller
    |
14  |             let webview_window = WebviewWindowBuilder::from_config(handle, &app.config().app.windows.get(0).unwrap().clone())?
    |                                                                                                                              +
```

--- a/.changes/allow-downloads-default.md
+++ b/.changes/allow-downloads-default.md
@@ -1,0 +1,5 @@
+---
+wry: patch
+---
+
+Enabled all downloads by default to match WebView2 and browser behavior on all platforms. To disable this, provide a custom `download_started_handler`.

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -585,6 +585,8 @@ pub struct WebViewAttributes<'a> {
   /// second is a mutable `PathBuf` reference that (possibly) represents where the file will be downloaded to. The latter
   /// parameter can be used to set the download location by assigning a new path to it, the assigned path _must_ be
   /// absolute. The closure returns a `bool` to allow or deny the download.
+  ///
+  /// [`Self::default()`] sets a handler allowing all downloads to match browser behavior.
   pub download_started_handler: Option<Box<dyn FnMut(String, &mut PathBuf) -> bool + 'static>>,
 
   /// A download completion handler to manage downloads that have finished.
@@ -718,7 +720,7 @@ impl Default for WebViewAttributes<'_> {
       ipc_handler: None,
       drag_drop_handler: None,
       navigation_handler: None,
-      download_started_handler: None,
+      download_started_handler: Some(Box::new(|_, _| true)),
       download_completed_handler: None,
       new_window_req_handler: None,
       clipboard: false,
@@ -1159,6 +1161,8 @@ impl<'a> WebViewBuilder<'a> {
   /// second is a mutable `PathBuf` reference that (possibly) represents where the file will be downloaded to. The latter
   /// parameter can be used to set the download location by assigning a new path to it, the assigned path _must_ be
   /// absolute. The closure returns a `bool` to allow or deny the download.
+  ///
+  /// By default a handler that allows all downloads is set to match browser behavior.
   pub fn with_download_started_handler(
     mut self,
     download_started_handler: impl FnMut(String, &mut PathBuf) -> bool + 'static,

